### DISCUSSION TITLE
Duda con los Fondos en CSS

### DISCUSSION THREAD

**Discussion by MitchelAlvarez** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#225] [Q&A]
Que tal compañeros, 
Sobre el tema de los fondo quisiera sabes porque cuando pongo texto es section o el div, el fondo se pierde un poco de la parte de arriba y queda en blanco(ver imegen adjunta).
Alguien sabra como puedo hacerle para que es parte sea completa mente del color y yo poder escribir?
Si lo menciono en el live pero no si se puede arreglar o no.
Saludos y gracias

![duda1](https://user-images.githubusercontent.com/99301935/156670623-d3eb38e4-af8a-4c7e-b46d-7fd4078848c9.PNG)

**Comment by enriqueorea** [comment]
si pudieras compartir el código te podría ayudar un poco más, pero por lo que creo debe ser problemas con la propiedad de Margin en alguna etiqueta, por ejemplo el navegador agrega por defecto margenes a las etiquetas h y p

**Reply by enriqueorea** [reply]
+Debes de utilizar algún reset de CSS o crear el tuyo, recuerda que por defecto el navegador le da estilos a las etiquetas

**Comment by hackit92** [comment]
Por defecto los navegadores tienen ciertas propiedades CSS, si no le asignas un valor de margin o padding específico podría añadir los valores por defecto a tus divs.

**Comment by Aragui** [comment]
No estoy muy seguro como tengas tu css, pero el navegador le mete unos padding al body, asi que para eliminar esos espacios en blanco tienes que quitarselos declarando body{ padding: 0; }

**Comment by davidtoString** [comment]
Me parece que es porque tienes dos ```<div>``` , uno de verde y otro de café.
Si lo tienes así: 

```
<div>
<p>Contenido en verde</p>
</div> 
<div>
<p>Contenido en café</p>
</div>
```


Prueba cambiarlo por

```
<div>
<p>Contenido en verde</p>
</div><div>
<p>Contenido en café</p>
</div>
```

Sin cambio de linea al terminar un div y comenzar el otro. 


O como alternativa, ponle:

`float:left;
clear:none;  `

a ambos divs.

**Comment by MitchelAlvarez** [comment]
Gracias a todos  ya intente lo de padding 0 y lo de poner los div o section sin salto de linea y sigue igual :( 
El codigo es el del profe de fondos: y le agrege los <h1>
```
<section class="fondoColor">
        <h1>test</h1>
    </section>
    <div class="fondoOpacidad">
        <h1>Pasteleria</h1>
    </div>
```

CSS:
```
*{
    font-family: Arial, Helvetica, sans-serif;
}

section{
    height: 400px;
    padding: 0%;
}
```

**Reply by enriqueorea** [reply]
ya ya mira, en el css agrega h1{margin 0; padding:0;}

**Reply by enriqueorea** [reply]
a mi gusto es mala practica pero en el asterisco agrega margin: 0, y padding:0;

**Reply by MitchelAlvarez** [reply]
Muchísimas gracias si funciono así y se ve como quería, por que es mala practica?

**Reply by enriqueorea** [reply]
porque el asterisco hace referencia a TODAS las etiquetas y puede ser que pases de largo alguna que tal vez no querias, claro más adelante en el css lo puedes sobre escribir. cuestión de forma de trabajo

**Reply by MitchelAlvarez** [reply]
muchísimas gracias, lo tomare en cuenta :)

**Comment by Ingrzs** [comment]
La razon es simple todo componente tiene su padding, igual su margin , 
Para que veas donde empieza el otro y dónde termina...
Cuando ya avanzes mas en el tema resetea el navegador para que todo el espacio ya que por default el nabegador "roba espacio"
